pkg/tsdb/opentsdb/types.go

```diff
@@ -7,7 +7,7 @@ type OpenTsdbQuery struct {
 }
 
 type OpenTsdbResponse struct {
-	Metric     string             `json:"metric"`
-	Tags       map[string]string  `json:"tags"`
-	DataPoints map[string]float64 `json:"dps"`
+	Metric     string            `json:"metric"`
+	Tags       map[string]string `json:"tags"`
+	DataPoints [][]float64       `json:"dps"`
```

Comment:
I'm not why this was changed to a double array instead of using a map

Reply:
That's the new data structure that's returned from OpenTSDB now